test/hook.test.js

```diff
@@ -5,6 +5,8 @@ const noop = () => {}
 const plugin = require('../').internals.factory
 const key = '123456789012354579814'
 const keys = { keys: new Set([key]) }
+const bearerAlt = 'BearerAlt'
+const keysAlt = { keys: new Set([key]), bearerType: bearerAlt }
```

Comment:
Why is this a top level variable? Shouldn’t the config be defined in the alt specific test?